keps/1432-admission-check-per-resource-flavor/kep.yaml

```diff
@@ -0,0 +1,41 @@
+title: AdmissionCheck per ResourceFlavor
+kep-number: 1913
+authors:
+  - "@pbundyra"
+status: implementable
+creation-date: 2024-03-26
+reviewers:
+  - "@mimowo"
+  - "@gabesaba"
```

Comment:
I think @mwielgus wanted to review this one.

@tenzen-y do you have anything to say about this feature?

Reply:
Let me check.